demos/demos.css

```diff
@@ -56,6 +56,7 @@ fieldset {
   align-items: center;
   justify-content: center;
   flex-flow: row nowrap;
+  height:360px;
```

Comment:
Sorry to bother you, but I'm having a bit of trouble getting the tests to run. Everything runs fine through the karma tests that open up a chrome instance, but the next test fails with 
`
> @ test:closure C:\Users\axh12\Documents\material-components-web
> ./scripts/closure-test.sh

'.' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "test:closure"
npm ERR! node v6.9.5
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! @ test:closure: `./scripts/closure-test.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ test:closure script './scripts/closure-test.sh'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./scripts/closure-test.sh
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\axh12\Documents\material-components-web\npm-debug.log
npm ERR! Test failed.  See above for more details.
`
I'm trying to run 'npm test' with the version of bash that git comes with, but it's still struggling to read that .sh file. I might end up just creating a linux VM, but I was wondering if you might have any additional ideas first. Thank you!